### ISSUE TITLE
mv tldr page disagrees with the mv man page

### ISSUE THREAD

**Issue by arunisaac** [tldr-pages/tldr#2481]
The tldr page for the mv command (https://github.com/tldr-pages/tldr/blob/master/pages/common/mv.md) specifies that the -i flag make mv not prompt for confirmation, but print a warning to stderr. However, according to the GNU coreutils man page for mv (https://linux.die.net/man/1/mv), the -i flag is to enable interactive mode and causes mv to prompt before confirmation. Thus the mv tldr page and the mv man page seem to be inconsistent. The mv tldr page is likely incorrect.

**Comment by agnivade** [comment]
Thanks for catching this. A PR will be most welcome :)

**Comment by arunisaac** [comment]
Should I just delete the third example (the one about mv -fi) in the mv
tldr page?

Also, I think the second example (the one about mv -f) is also
misleading because mv, by default without any flags, does not prompt for
confirmation. There is no need to supply the -f flag. Should I delete
this example too?
